### DISCUSSION TITLE
Removal of unique_version_history impact to meta and keyval resource types

### DISCUSSION THREAD

**Discussion by gaigaslab-operations** [concourse/concourse#6594] [Help & Support]
With the removal of unique_version_history from resource type definitions as implemented in https://github.com/concourse/concourse/pull/6022, the meta and keyval resource types have become global, which defeats their purpose.

Is there a work-around other than global unique naming of the resource?

**Comment by gaigaslab-operations** [comment]
Bump

**Comment by vito** [comment]
Aren't these meant to be used by running a `put` which generates a unique version every time? I don't think that should be affected by sharing history.

**Comment by gaigaslab-operations** [comment]
We use them so that the current pipeline job/task understands what its
pipeline/job/task is.  If the values are shared and overwritten when
another task starts, then the values crease to represent reality.
